### ISSUE TITLE
Imagery: everything is too dark and over-saturated

### ISSUE THREAD

**Issue by tillgrallert** [oibeirut/oib_website#2]
There is an automatic filter applied to all images. Currently, I don't know whether this occurs upon upload or during rendering. However, the effect is devastating. In the people section, images become black squares ([example](https://www.orient-institut.org/people/fellows/)). Upon hovering above images on the same page, they become over-saturated.

**Comment by GeorgeBatrouni** [comment]
This is done, as for the images they have a hover effect which makes them black and white, we can remove it completely if you don't like it

**Comment by tillgrallert** [comment]
Dear @GeorgeBatrouni, I would not remove the hovering effect. As long as the automatic processing for saturation and contrast upon upload has been removed.

**Comment by DavidKattanOIB** [comment]
IMHO, the hovering effect can still be used even if the over-saturation/high contrast issues are solved. What do you think @tillgrallert @patrick-mzaaber ?

**Comment by patrick-mzaaber** [comment]
> IMHO, the hovering effect can still be used even if the over-saturation/high contrast issues are solved. What do you think @tillgrallert @patrick-mzaaber ?

Yes, dear is better to have the hovering effect for both reasons focus and style  @DavidKattanOIB

**Comment by GeorgeBatrouni** [comment]
Hello, This is done, we fixed the hover effect and reduced the over saturation and high contrast
